App crashes at start on current master, beta debug

**Summary:** 
I fetched upstream and app does not start anymore. After being logged in it crashes.

**Steps to reproduce:** 
Start the app with current master

**System logs:**
```
2019-11-28 15:11:54.309 31899-31899/fr.free.nrw.commons.beta E/ACRA: ACRA caught a RuntimeException for fr.free.nrw.commons.beta
    java.lang.RuntimeException: Unable to create service fr.free.nrw.commons.upload.UploadService: java.lang.IllegalArgumentException: baseUrl must end in /: https://www./wikidata.org
        at android.app.ActivityThread.handleCreateService(ActivityThread.java:3414)
        at android.app.ActivityThread.-wrap4(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1683)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6541)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
     Caused by: java.lang.IllegalArgumentException: baseUrl must end in /: https://www./wikidata.org
        at retrofit2.Retrofit$Builder.baseUrl(Retrofit.java:515)
        at retrofit2.Retrofit$Builder.baseUrl(Retrofit.java:458)
        at org.wikipedia.dataclient.ServiceFactory.createRetrofit(ServiceFactory.java:61)
        at org.wikipedia.dataclient.ServiceFactory.get(ServiceFactory.java:39)
        at fr.free.nrw.commons.di.NetworkingModule.provideWikidataInterface(NetworkingModule.java:216)
        at fr.free.nrw.commons.di.NetworkingModule_ProvideWikidataInterfaceFactory.proxyProvideWikidataInterface(NetworkingModule_ProvideWikidataInterfaceFactory.java:35)
        at fr.free.nrw.commons.di.NetworkingModule_ProvideWikidataInterfaceFactory.get(NetworkingModule_ProvideWikidataInterfaceFactory.java:24)
        at fr.free.nrw.commons.di.NetworkingModule_ProvideWikidataInterfaceFactory.get(NetworkingModule_ProvideWikidataInterfaceFactory.java:10)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at fr.free.nrw.commons.wikidata.WikidataClient_Factory.get(WikidataClient_Factory.java:16)
        at fr.free.nrw.commons.wikidata.WikidataClient_Factory.get(WikidataClient_Factory.java:7)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at fr.free.nrw.commons.wikidata.WikidataEditService_Factory.get(WikidataEditService_Factory.java:35)
        at fr.free.nrw.commons.wikidata.WikidataEditService_Factory.get(WikidataEditService_Factory.java:9)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at fr.free.nrw.commons.di.DaggerCommonsApplicationComponent$UploadServiceSubcomponentImpl.injectUploadService(DaggerCommonsApplicationComponent.java:3095)
        at fr.free.nrw.commons.di.DaggerCommonsApplicationComponent$UploadServiceSubcomponentImpl.inject(DaggerCommonsApplicationComponent.java:3090)
        at fr.free.nrw.commons.di.DaggerCommonsApplicationComponent$UploadServiceSubcomponentImpl.inject(DaggerCommonsApplicationComponent.java:3079)
        at dagger.android.DispatchingAndroidInjector.maybeInject(DispatchingAndroidInjector.java:113)
        at dagger.android.DispatchingAndroidInjector.inject(DispatchingAndroidInjector.java:134)
        at fr.free.nrw.commons.di.CommonsDaggerService.inject(CommonsDaggerService.java:28)
        at fr.free.nrw.commons.di.CommonsDaggerService.onCreate(CommonsDaggerService.java:15)
        at fr.free.nrw.commons.HandlerService.onCreate(HandlerService.java:55)
        at fr.free.nrw.commons.upload.UploadService.onCreate(UploadService.java:119)
```

**Device and Android version:** 

API 21 Emulator
 
**Commons app version:** 
betaDebug

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -222,7 +222,7 @@ android {
             buildConfigField "String", "IMAGE_URL_BASE", "\"https://upload.beta.wmflabs.org/wikipedia/commons\""
             buildConfigField "String", "HOME_URL", "\"https://commons.wikimedia.beta.wmflabs.org/wiki/\""
             buildConfigField "String", "COMMONS_URL", "\"https://commons.wikimedia.beta.wmflabs.org\""
-            buildConfigField "String", "WIKIDATA_URL", "\"https:/www./wikidata.org\""
+            buildConfigField "String", "WIKIDATA_URL", "\"https://www.wikidata.org\""
             buildConfigField "String", "MOBILE_HOME_URL", "\"https://commons.m.wikimedia.beta.wmflabs.org/wiki/\""
             buildConfigField "String", "SIGNUP_LANDING_URL", "\"https://commons.m.wikimedia.beta.wmflabs.org/w/index.php?title=Special:CreateAccount&returnto=Main+Page&returntoquery=welcome%3Dyes\""
             buildConfigField "String", "SIGNUP_SUCCESS_REDIRECTION_URL", "\"https://commons.m.wikimedia.beta.wmflabs.org/w/index.php?title=Main_Page&welcome=yes\""

